### PR TITLE
Added Mish activation

### PR DESCRIPTION
**Pull request by digantamisra98**
Referring to issue - [144](https://github.com/FluxML/NNlib.jl/issues/144)

### CODE REVIEW

**Comment by bhvieira** [comment]
This would be nice to have, +1!

**Comment by staticfloat** [comment]
Looks good to me; can you add some simple tests, (take a look at the tests in https://github.com/FluxML/NNlib.jl/blob/master/test/activation.jl#L123-L141 for a simple example) so that we can make sure we don't accidentally break it in the future?

**Comment by digantamisra98** [comment]
@staticfloat Hi, thanks for the review. Have added the tests.

File: test/activation.jl [hunk 1]
```diff
@@ -127,6 +130,12 @@ end
         @test elu(-4) ≈ (exp(-4) - 1)
     end
 
+    @testset "mish" begin
+        @test mish(-5) ≈ -0.033576237730161704
+        @test mish(9) == 9*tanh(log(1 + exp(9)))
+        xs = Float32[1 2 3; 1000 2000 3000]
+        @test typeof(mish.(xs)) == typeof(xs)
```
Comment (staticfloat) [inline test/activation.jl]: I think you're missing an `end` here. :)

Comment (digantamisra98) [reply test/activation.jl]: Fixed. Thanks!

**Comment by staticfloat** [comment]
Fantastic!  Thanks!

**Pull request merged**